uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -820,6 +843,31 @@ public void discoverCoreDeviceBroker(String brokerId) {
         log.info("Added IoT Core broker as {} with endpoint {}:{}", brokerId, endpoint, IOT_CORE_MQTT_PORT);
     }
 
+
+    /**
+     * Set up IoT core broker port.
+     *
+     * @param brokerId broker name in tests
+     * @param port broker port in tests
+     */
+    @And("I force to set IoT Core broker as {string} with port {int}")
+    public void setBrokerPort(String brokerId, int port) {
+        final String endpoint = resources.lifecycle(IotLifecycle.class)
+                                         .dataEndpoint();
+        final String ca = registrationContext.rootCA();
+        List<MqttBrokerConnectionInfo> connectionInfos = brokers.get(brokerId);
+        if (connectionInfos != null && !connectionInfos.isEmpty()) {
+            for (MqttBrokerConnectionInfo info : connectionInfos) {
+                info.setPort(port);
+            }
+        } else {
+            MqttBrokerConnectionInfo broker = new MqttBrokerConnectionInfo(
+                    endpoint, port, Collections.singletonList(ca));
+            brokers.put(brokerId, Collections.singletonList(broker));
+        }
+        log.info("Added IoT Core broker as {} with endpoint {}:{}", brokerId, endpoint, port);
```

Comment:
Port of broker AAA force changed to INT

Reply:
changed in a39a3bb388a83630e9a3cf4c44a52ab0c2c8026e